readme.md

```diff
@@ -82,13 +86,32 @@ require('@zeit/ncc')('/path/to/input', {
   v8cache: false, // default
   quiet: false, // default
   debugLog = false // default
-}).then(({ code, map, assets }) => {
-  console.log(code);
-  // Assets is an object of asset file names to { source, permissions, symlinks }
-  // expected relative to the output code (if any)
+}).then(({ output }) => {
+  // Output is an object of file names to { source, permissions } | string
+  // Where string entries represent symlinks to other files, referenced relatively.
+  // The main build file is located at output[filename].
+  output['build.js'].source
 })
 ```
 
+Multiple entry points can be built by providing an object to build. In this case the output returned is an array of `{ code, map }` pairs:
```

Comment:
This doesn't seem correct. There is no `code` anymore and `map` seems to be part of the `output` as you mentioned below.

Reply:
Thanks.